primitives/io/src/lib.rs

```diff
@@ -99,12 +101,22 @@ pub enum EcdsaVerifyError {
 /// removed from the backend from making the `storage_kill` call.
 #[derive(PassByCodec, Encode, Decode)]
 pub enum KillStorageResult {
-	/// All key to remove were removed, return number of key removed from backend.
+	/// All keys to remove were removed, return number of key removed from backend.
 	AllRemoved(u32),
 	/// Not all key to remove were removed, return number of key removed from backend.
```

Comment:
Per discussion, the docs should instead say the following:
```suggestion
	/// All keys to remove were removed, return number of iterations performed during the operation.
	AllRemoved(u32),
	/// Not all key to remove were removed, return number of iterations performed during the operation.
```